src/Action.hs

```diff
@@ -468,12 +501,37 @@ markIdeaInResultPhase iid rv = do
 
 -- * Topic handling
 
-topicInRefinementTimedOut :: (ActionPersist m, ActionSendMail m) => AUID Topic -> m ()
+topicInRefinementTimedOut :: (ActionPhaseChange m) => AUID Topic -> m ()
 topicInRefinementTimedOut = topicTimeout RefinementPhaseTimeOut
 
-topicInVotingTimedOut :: (ActionPersist m, ActionSendMail m) => AUID Topic -> m ()
+topicInVotingTimedOut :: (ActionPhaseChange m) => AUID Topic -> m ()
 topicInVotingTimedOut = topicTimeout VotingPhaseTimeOut
 
+topicInVotingResetToJury
+    :: (ActionPhaseChange m) => AUID Topic -> m ()
+topicInVotingResetToJury tid = do
+    topic <- mquery $ findTopic tid
+    case topic ^. topicPhase of
+        PhaseVoting _ -> topicPhaseChange topic VotingPhaseSetbackToJuryPhase
+        _             -> pure ()
+
+-- * Admin activities
+
+-- | Make a topic timeout if the timeout is applicable.
+-- FIXME: Only admin can do that
```

Comment:
Isn't this now assured via capabilities in `LifeCycle`? Hmm, now that I look at it, `ideaCapabilities` seems barely used --- is it that it needs to be queried somewhere before the user is allowed to call `topicForceNextPhase`?